WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/ai/productinfo/AiProductPromptScreen.kt

```diff
@@ -339,49 +341,58 @@ private fun ProductPromptTextField(
+                                coroutineScope.launch {
+                                    @Suppress("MagicNumber")
+                                    delay(200) // Delay to ensure advice box is shown before scrolling.
+                                    scrollState.animateScrollTo(scrollToPosition.roundToInt())
+                                }
                             }
                         }
                 )
             }
-
-            Divider()
-
-            when {
-                state.isScanningImage -> ImageScanning()
-                state.selectedImage != null -> SelectedImageSection(
-                    image = state.selectedImage,
-                    onImageActionSelected = onImageActionSelected,
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .background(color = colorResource(id = R.color.ai_generated_text_background))
-                )
-
-                else -> {
-                    WCTextButton(
-                        modifier = Modifier
-                            .fillMaxSize()
-                            .padding(horizontal = dimensionResource(id = R.dimen.minor_50)),
-                        onClick = onReadTextFromProductPhoto,
-                        icon = ImageVector.vectorResource(id = R.drawable.ic_gridicons_camera_primary),
-                        allCaps = false,
-                        text = stringResource(id = R.string.ai_product_creation_read_text_from_photo_button),
-                    )
-                }
-            }
         }
         AnimatedVisibility(
             (isFocused || state.productPrompt.isNotEmpty()) &&
                 FeatureFlag.PRODUCT_CREATION_WITH_AI_V2_M3.isEnabled()
         ) {
             PromptSuggestions(
                 promptSuggestionBarState = state.promptSuggestionBarState,
-                modifier = Modifier.padding(top = 16.dp)
+                modifier = Modifier.padding(top = 8.dp)
             )
         }
     }
 }
 
+@Composable
+private fun ScanProductPhotoButton(
+    state: AiProductPromptState,
+    onReadTextFromProductPhoto: () -> Unit,
+    onImageActionSelected: (ImageAction) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    when {
+        state.isScanningImage -> ImageScanning()
+        else -> SelectImageSection(
+            image = state.selectedImage,
+            onImageActionSelected = onImageActionSelected,
+            onReadTextFromProductPhoto = onReadTextFromProductPhoto,
+            subtitle = when {
+                state.selectedImage == null -> stringResource(id = R.string.ai_product_creation_image_scan_subtitle)
+                else -> stringResource(id = R.string.ai_product_creation_image_selected_subtitle)
+            },
+            modifier = modifier
+                .fillMaxWidth()
+                .clip(RoundedCornerShape(dimensionResource(id = R.dimen.minor_100)))
```

Comment:
np, it would be great to stick to literal dimensions for consistency
```suggestion
                .clip(RoundedCornerShape(8.dp))
```